### PR TITLE
Changed date for synthetic events

### PR DESCRIPTION
**Pull request by lamagnifica**
@brianhw, @catong fixes DOC-1320
Change (date in Reference.rst) has already been released.
@srpearce , @mhoeber FYI (same changes to preface)

### CODE REVIEW

File: docs/en_us/shared/preface.rst [hunk 1]
```diff
@@ -65,7 +73,9 @@ Documentation for course teams is available on the docs.edx.org web page.
   peer- and self- evaluations of responses to a question. Note that this new
   feature is in limited release.
 
-* `edX Open Learning XML Guide`_ provides guidelines for building edX courses with Open Learning XML (OLX). Note that this guide is currently an Alpha version.
+* `edX Open Learning XML Guide`_ provides guidelines for building edX courses
+  with Open Learning XML (OLX). Note that this guide is currently an Alpha
```
Comment (srpearce) [inline docs/en_us/shared/preface.rst]: Not sure why "alpha" is capped - ?

File: docs/en_us/shared/preface.rst [hunk 2]
```diff
@@ -223,8 +233,8 @@ interact with other students and with the course team: click **Discussion**.
 Many courses also offer a wiki for additional resources and materials: click
 **Wiki**.
 
-Other resources may also be available, such as a course-specific facebook page
-or twitter feed or opportunites for Google hangouts. Be sure to check the
+Other resources may also be available, such as a course-specific Facebook page
+or Twitter feed, or opportunities for Google hangouts. Be sure to check the
```
Comment (srpearce) [inline docs/en_us/shared/preface.rst]: "Hangouts" should be capped.

**Comment by catong** [comment]
Minor edits, then :+1:

File: docs/en_us/shared/preface.rst (hunk 2 above)
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst]: done

**Comment by mhoeber** [comment]
:+1:

**Comment by lamagnifica** [comment]
@brianhw , waiting on your review

**Comment by lamagnifica** [comment]
Brian says :+1:

**Pull request merged**